Modules/Core/Common/test/itkNeighborhoodOperatorTest.cxx

```diff
@@ -77,45 +78,82 @@ println(const char * c)
 int
 itkNeighborhoodOperatorTest(int, char *[])
 {
-  println("Testing derivative operator");
+  std::cout << "Testing derivative operator" << std::endl;
   itk::DerivativeOperator<float, 3, vnl_vector<float>> d;
```

Comment:
```c++
using Real = float;
constexpr unsigned int Dimension = 3;
itk::DerivativeOperator<Real, Dimension, vnl_vector<Real>> d;
```
... and use `Real` and/or `Dimension` in some line below as well.  (Or skip my `Dimension` suggestion because other values for Dimension are used later.)

Reply:
Used an approach inspired by the suggested snippet in 8449d1a.